package.json

```diff
@@ -11,14 +13,15 @@
     "node": ">=10"
   },
   "scripts": {
-    "build": "tsc",
+    "clean": "rm -rf ./dist",
+    "build": "tsup src/index.ts --dts --sourcemap --target es5 --format cjs,esm",
```

Comment:
Could we make this work with TSC itself, too? We can't drop the Node 10 CI.

Reply:
I'll take a look and see if I can achieve the same output using `tsc` alone.

Have in mind that it's only building in Node 10 that fails, using the library in Node 10 should work.

Reply:
Added a fallback to `tsc` in Node 10 since `tsup` uses the `TextEncoder` global which was added in Node 12.

Reply:
Wonder why Node.js 10 support is still required, it's reaching the end of life next month

Reply:
(just removed TextDecoder from tsup since it's not necessary)

Reply:
Node.js is pre-ES6 and thus a good proxy for testing legacy browser support.